docs/gadgets/seccomp.md

```diff
@@ -12,12 +12,14 @@ kind: Trace
 metadata:
   name: seccomp
   namespace: gadget
+  labels:
+    trace-instance: "test123"
+    trace-node: "minikube"
 spec:
   node: minikube
   gadget: seccomp
   filter:
-    namespace: kube-system
-    podname: etcd-minikube
+    namespace: default
```

Comment:
Is the podname not needed anymore? Is it replaced by the trace-instance label? :-/ I find this rather confusing...